toke.c

```diff
@@ -606,26 +606,6 @@ S_missingterm(pTHX_ char *s, STRLEN len)
 
 #include "feature.h"
 
-/*
- * Check whether the named feature is enabled.
- */
-bool
-Perl_feature_is_enabled(pTHX_ const char *const name, STRLEN namelen)
```

Comment:
note that we will also have to remove or adjust `feature_is_enabled` from Level::PPPort

Reply:
Probably remove, feature_is_enabled was only used by FEATURE_IS_ENABLED(), which is only intended for use by core or included extensions (PERL_EXT).